spec/requests/metrics_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+RSpec.describe 'Users', type: :request do
+  let(:admin) { create(:admin_user) }
+
+  describe 'root' do
+    context 'no facets provided' do
+      before { get "/v3/metrics" }
+
+      it 'returns user info' do
+        response_attributes = JSON.parse(response.body)
+
+        expect(response_attributes).to eq ({ 'errors' => 'facets parameter is required' })
+      end
+    end
+  end
+
+  # describe 'facets' do
```

Comment:
This story only covers request specs for stories. Metrics will be covered in another story.